src/savegame_legacy.cpp

```diff
@@ -1907,7 +1907,7 @@ void item::load_legacy(std::stringstream & dump) {
         }
     }
 
-    dump >> burnt >> poison >> ammotmp >> owned >> bday >>
+    dump >> burnt >> poison >> ammotmp >> bday >>
```

Comment:
You may want to be careful about changing this here, it looks like it may mess up legacy saves.